src/tests/ascent/t_ascent_slice.cpp

```diff
@@ -816,6 +816,330 @@ TEST(ascent_slice, test_auto_slice_xy_axis)
+                                              EXAMPLE_MESH_SIDE_DIM,
+                                              data);
+
+    EXPECT_TRUE(conduit::blueprint::mesh::verify(data,verify_info));
+    ASCENT_INFO("Testing sphere slice");
+
+    string output_path = prepare_output_dir();
+    string output_file = conduit::utils::join_file_path(output_path,"tout_slice_sphere_3d");
+
+    // remove old images before rendering
+    remove_test_image(output_file);
+
+    //
+    // Create the actions.
+    //
+    conduit::Node actions;
+    conduit::Node &add_pipelines = actions.append();
+    add_pipelines["action"] = "add_pipelines";
+    conduit::Node & pipelines = add_pipelines["pipelines"];
+    pipelines["pl1/f1/type"] = "slice";
+    conduit::Node &slice_params = pipelines["pl1/f1/params"];
+
+    slice_params["sphere/center/x"] = 0.0;
+    slice_params["sphere/center/y"] = 0.0;
+    slice_params["sphere/center/z"] = 0.0;
+    slice_params["sphere/radius"] = 10.0;
+
+    // add a scene
+    conduit::Node &add_scenes= actions.append();
+    add_scenes["action"] = "add_scenes";
+    conduit::Node &scenes = add_scenes["scenes"];
+    scenes["s1/plots/p1/type"]  = "pseudocolor";
+    scenes["s1/plots/p1/field"] = "radial";
+    scenes["s1/plots/p1/pipeline"] = "pl1";
+    scenes["s1/image_prefix"] = output_file;
+
+    // add an extract
+    conduit::Node &add_extracts= actions.append();
+    add_extracts["action"] = "add_extracts";
+    conduit::Node &extracts = add_extracts["extracts"];
+    extracts["e1/type"]  = "relay";
+    extracts["e1/pipeline"]  = "pl1";
+    extracts["e1/params/path"] = output_file + "_hdf5";
+    extracts["e1/params/protocol"] = "hdf5";
+
+    // run ascent
+    Ascent ascent;
+    ascent.open();
+    ascent.publish(data);
+    ascent.execute(actions);
+    ascent.close();
+
+    // check that we created an image
+    EXPECT_TRUE(check_test_image(output_file));
+    std::string msg = "An example of a spherical slice.";
+    ASCENT_ACTIONS_DUMP(actions,output_file,msg);
+}
+
+
+//-----------------------------------------------------------------------------
+TEST(ascent_slice, test_cyln_slice)
+{
+    Node n;
+    ascent::about(n);
+    // only run this test if ascent was built with vtkm support
+    if(n["runtimes/ascent/vtkm/status"].as_string() == "disabled")
+    {
+        ASCENT_INFO("Ascent vtkm support disabled, skipping test");
+        return;
+    }
+
+    //
+    // Create an example mesh.
+    //
+    Node data, verify_info;
+    conduit::blueprint::mesh::examples::braid("hexs",
+                                              EXAMPLE_MESH_SIDE_DIM,
+                                              EXAMPLE_MESH_SIDE_DIM,
+                                              EXAMPLE_MESH_SIDE_DIM,
+                                              data);
+
+    EXPECT_TRUE(conduit::blueprint::mesh::verify(data,verify_info));
+    ASCENT_INFO("Testing sphere slice");
+
```

Comment:
copy-pasta error: should be "Testing cylinder slice" 

Same goes for the other new ASCENT_INFO calls: "Testing box slice" "Testing plane slice"